mobile/src/main/java/org/openhab/habdroid/core/OpenHABTracker.java

```diff
@@ -95,7 +95,7 @@ public void start() {
                     // If remote URL is configured
                     if (mOpenHABUrl.length() > 0) {
                         Log.d(TAG, "Connecting to remote URL " + mOpenHABUrl);
-                        openHABTracked(mOpenHABUrl, mCtx.getString(R.string.info_conn_rem_url));
```

Comment:
I find the notifications anoying. When I'm in my wifi it connects to local url, if not to the remote url. I don't need feedback for that.
Every time I open the app I get these notifications and I get the "voice recognizer..." every time I rotate the phone